os-depends

```diff
@@ -130,6 +130,7 @@ xdg-user-dirs
 xdg-user-dirs-gtk
 xserver-xorg
 xserver-xorg-input-libinput
+xserver-xorg-input-wacom
```

Comment:
Not so sure about this one. https://phabricator.endlessm.com/T14678#663901

Reply:
Ah, I should have looked into this some more. My outdated understanding was that basically the only two X input drivers you want are libinput and wacom, but I think that was from before wacom support was added to libinput. I can update.